### DISCUSSION TITLE
Bash script to sync between Telekasten todos  and Mac/iPhone Reminders application

### DISCUSSION THREAD

**Discussion by cnshsliu** [nvim-telekasten/telekasten.nvim#221] [General]
I use Macbook pro and iPhone, normally the todos in  Reminders are synced between Mac/iPhone by iCloud.

However, I also create todos in Telekasten notes, such like take some notes while programming, think of something need to do, then I create todo in Telekasten notes directly.  When I walk away from Macbook, I'd like to check todos on iPhone. or, maybe just have something to remember before sleeping,  make one reminder in iPhone Reminders, then tomorrow morning when I come back to office, I could see it in Telekasten.

I wrote this script to sync todos between Telekasten and Mac Reminders bidirectionally.
so I am able to manage/check todos anywhere with MacBook or iPhone.

Share to you in case of this is also your routine.

https://github.com/cnshsliu/synctodo

**Comment by lambtho12** [comment]
Awesome! Thank you very much. Just added [a link to the README](https://github.com/renerocksai/telekasten.nvim#other-useful-resourcesplugins).

**Reply by cnshsliu** [reply]
Thanks a lot.
